SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/InterestManager.cpp

```diff
@@ -7,7 +7,12 @@
 #include <algorithm>
 #include <cmath>
 
+// Disabled until we find how to make AVX intrinsics work on linux + android.
+#define ENABLE_AVX 0
```

Comment:
Can we not just check against appropriate platform? e.g. `#ifdef PLATFORM_WINDOWS`

Reply:
or use the thing that epic use with the x86 platform

Reply:
This is removed now, now using PLATFORM_ANDROID specifically